Cargo.toml

```diff
@@ -27,13 +27,28 @@ harness = false
 name = "smt"
 harness = false
 
+[[bench]]
+name = "smt-subtree"
+harness = false
+required-features = ["internal"]
+
+[[bench]]
+name = "merkle"
+harness = false
+
+[[bench]]
+name = "smt-with-entries"
+harness = false
+
 [[bench]]
 name = "store"
 harness = false
 
 [features]
-default = ["std"]
+concurrent = ["dep:rayon"]
+default = ["std", "concurrent"]
 executable = ["dep:clap", "dep:rand-utils", "std"]
+internal = ["concurrent"]
```

Comment:
Why does `internal` need to imply `concurrent`?

Reply:
Good point - I introduced it during testing as part of an earlier setup, but I’ve cleaned it up now to remove the redundancy.